src/chemistry/aerosol/cloud_aqueous_chemistry.F90

```diff
@@ -0,0 +1,894 @@
+!----------------------------------------------------------------------------------
+! Cloud aqueous chemistry
```

Comment:
@tilmes @marybarthbrock - Could we get a high-level description of the purpose of this module in science terms?